.changeset/large-hounds-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Fix nested connection filters using SINGLE and SOME
```

Comment:
Shouldn't there just be one changeset for this PR?

Reply:
There are 2 issue fixes here, so I went with 2 changesets, not sure what the appropriate way of doing that is :/

Reply:
I _think_ the way we use changesets associates one changeset with one PR - so I would document that this PR addresses two issues with a single PR.

Reply:
So, two lines in this changeset and remove the other one then?

Reply:
```suggestion
Fix nested connection filters using SINGLE and SOME
Fix implicit and parameters missing in connection where
```